.github/workflows/soroban-rpc.yml

```diff
@@ -105,13 +105,18 @@ jobs:
     continue-on-error: true
     strategy:
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04]
-        go: [1.22]
+        os: [ ubuntu-20.04, ubuntu-22.04 ]
+        go: [ 1.22 ]
+        protocol-version: [ 20, 21 ]
     runs-on: ${{ matrix.os }}
     env:
       SOROBAN_RPC_INTEGRATION_TESTS_ENABLED: true
+      SOROBAN_RPC_INTEGRATION_TESTS_CORE_MAX_SUPPORTED_PROTOCOL: ${{ matrix.protocol-version }}
       SOROBAN_RPC_INTEGRATION_TESTS_CAPTIVE_CORE_BIN: /usr/bin/stellar-core
-      PROTOCOL_20_CORE_DEBIAN_PKG_VERSION: 20.2.0-1716.rc3.34d82fc00.focal
+      PROTOCOL_20_CORE_DEBIAN_PKG_VERSION: 21.0.0-1812.rc1.a10329cca.focal
+      PROTOCOL_20_CORE_DOCKER_IMG: stellar/unsafe-stellar-core:21.0.0-1812.rc1.a10329cca.focal
```

Comment:
these values match the protocol 21 environment variables below, is that intentional?

Reply:
Yep, the protocol 21 should be able to run protocol 20 too